modules/app.xql

```diff
@@ -344,7 +344,11 @@ declare function app:browse-items($name as xs:string, $query as xs:string, $page
         for $key in map:keys($map)
             let $count := $map($key)
             let $percent := (math:log10($count) div $max)
-            let $output := if ($query = 'language') then lang:code2lang($key) else $key
+            let $output := 
+                switch($query)
```

Comment:
I always forget about `switch()` since it's an XQuery only thing! Much nicer.

Reply:
I always have to look up the syntax because, like everything else XQuery, it's just a little bit different.